schema/MTMount.yaml

```diff
@@ -26,20 +26,29 @@ properties:
     exclusiveMinimum: 0
     default: 10
   camera_cable_wrap_advance_time:
-      description: >-
-        How far in advance of the current time to make the tai time field of camera cable wrap
-        tracking commands (sec). All tracking commands must sent to the low-level controller in advance,
-        so it can interpolate between pairs of tracking command. If a tracking command is late then
-        the controller will halt the axis. Set this value large enough that camera cable wrap
-        commands reliably arrive in time, despite variation in speed of the Python CSC.
-        Avoid making it much larger than required, because it introduces a delay in motion
-        of the camera cable wrap, which can cause too large an error between the camera rotator
-        and the camera cable wrap when the rotator is offset or slewed to a new field.
-      type: number
-      default: 0.02
+    description: >-
+      How far in advance of the current time to make the tai time field of camera cable wrap
+      tracking commands (sec). All tracking commands must sent to the low-level controller in advance,
+      so it can interpolate between pairs of tracking command. If a tracking command is late then
+      the controller will halt the axis. Set this value large enough that camera cable wrap
+      commands reliably arrive in time, despite variation in speed of the Python CSC.
+      Avoid making it much larger than required, because it introduces a delay in motion
+      of the camera cable wrap, which can cause too large an error between the camera rotator
+      and the camera cable wrap when the rotator is offset or slewed to a new field.
+    type: number
+    default: 0.02
+  max_rotator_position_error:
+    description: >-
+      The maximum difference (in degrees) between camera rotator actual position and demand position
+      beyond which the camera cable wrap will follow the camera rotator actual position and velocity,
+      rather than the usual demand position and velocity.
+    type: number
+    exclusiveMinimum: 0
+    default: 0.1
 required:
   - host
   - connection_timeout
   - ack_timeout
   - camera_cable_wrap_advance_time
+  - max_rotator_position_error
```

Comment:
If they have defaults, do they need to be required?

Reply:
It seems safest to require them, but I am honestly not sure. It is what I have done in all my CSCs, but I can't remember if that was by force or by choice. I admit it does add some repetition, which is bad.